src/test/java/com/mongodb/kafka/connect/source/schema/AvroSchemaTest.java

```diff
@@ -67,13 +67,13 @@ void testSupportsValidAvroSchema() {
     Schema actual = AvroSchema.fromJson(schema);
 
     SchemaBuilder nodeBuilder =
-        SchemaBuilder.struct().name("Node").field("label", Schema.STRING_SCHEMA);
+        SchemaBuilder.struct().name("org.apache.avro.Node").field("label", Schema.STRING_SCHEMA);
```

Comment:
Now that we process namespaces, they appear in the parsed schemas, and we need to expect them.